testsuite/MDAnalysisTests/test_auxiliary.py

```diff
@@ -35,7 +35,12 @@ def __init__(self):
 class BaseAuxReaderTest(object):
     def __init__(self, reference):
         self.ref = reference
-        self.reader = self.ref.reader('test', self.ref.testdata)
+
+    def setUp(self):
```

Comment:
you can. The setup and teardown are called before each test in the class. I actually like using the `__init__` in the beginning for stuff that takes a long time as it speeds up the tests.